### ISSUE TITLE
Maintenance!

### ISSUE THREAD

**Issue by harisnaeemofficial** [PopcornRepos/PopcornTimeOnline#27]
Any one is maintaining this repo ?

**Comment by vaporwavie** [comment]
Uh, I don't think so. If you check the commits, the last one was approximately 2 years ago...

**Comment by ohhsodead** [comment]
There have been multiple editions since this project, it's probably still around somewhere

**Comment by harisnaeemofficial** [comment]
i have a version of this one but it stucks on downloading can any one help me about it if you know ?
http://www.harisnaeem.tk

**Comment by vaporwavie** [comment]
@harix786 I'm having this output when accessing your website: 
![image](https://user-images.githubusercontent.com/11232965/34412314-4496bdb8-ebbb-11e7-98c5-d35a744d9475.png)

**Comment by harisnaeemofficial** [comment]
@vaporwavie 
i am using a free hosting thats why my website was sleeping for some time from the server side . Now website is online

**Comment by harisnaeemofficial** [comment]
@vaporwavie 
See now working good
![screenshot 23](https://user-images.githubusercontent.com/24369463/34414853-71504f34-ec0d-11e7-87b3-fd8ef7851826.png)

But stucks on downloading at 0%
![screenshot 22](https://user-images.githubusercontent.com/24369463/34414867-8d0cca40-ec0d-11e7-9b34-97d0899eba05.png)

**Comment by vaporwavie** [comment]
Could you send me a screenshot of your console? Maybe some errors are appearing.

**Comment by harisnaeemofficial** [comment]
@vaporwavie 
Ok Send me you email  will give you all the related  details about it
 Here is the screen shots 
![screenshot 24](https://user-images.githubusercontent.com/24369463/34431705-2636a4ea-ec93-11e7-89b4-71a9f0525631.png)

and when stucks on 0% downloading it shows the magnet link and file format with name but dont know what is the problem in downloading
![screenshot 25](https://user-images.githubusercontent.com/24369463/34431732-4c082dc4-ec93-11e7-996a-d97e26e70c4e.png)

**Comment by vaporwavie** [comment]
You can find my e-mail right on my Github profile :) 

About the "stuck at 0%"... It might happen because some stuff of your project is actually configured on localhost. That may cause some bugs

**Comment by harisnaeemofficial** [comment]
Look at this http://popcorntime-online.tv
My website is same like this one and same errors are shown on this website but thats not the problem

**Comment by vaporwavie** [comment]
The required plugin is installed and everything, right?

**Comment by harisnaeemofficial** [comment]
yes i installed the torrents time twice but the problem is same 
when i visit this http://popcorntime-online.tv  site the downloading works fine

**Comment by harisnaeemofficial** [comment]
@vaporwavie 
Watch this 
![screenshot 31](https://user-images.githubusercontent.com/24369463/34440854-ef0d88a2-ecd9-11e7-9c77-c13203d26a90.png)
![screenshot 32](https://user-images.githubusercontent.com/24369463/34440856-ef49431a-ecd9-11e7-9eaf-0f016350ae97.png)
![screenshot 33](https://user-images.githubusercontent.com/24369463/34440857-ef82a0b0-ecd9-11e7-83f6-490232775e84.png)
![screenshot 34](https://user-images.githubusercontent.com/24369463/34440858-efb99e44-ecd9-11e7-96d7-9a79fd94e509.png)
![screenshot 35](https://user-images.githubusercontent.com/24369463/34440859-effdca6a-ecd9-11e7-9fbf-91535699abf4.png)
![screenshot 36](https://user-images.githubusercontent.com/24369463/34440860-f0476efe-ecd9-11e7-8afd-aee225612912.png)
![screenshot 37](https://user-images.githubusercontent.com/24369463/34440861-f0a0858e-ecd9-11e7-8346-fe25e6b36bc0.png)
![screenshot 38](https://user-images.githubusercontent.com/24369463/34440862-f0fcd7c6-ecd9-11e7-994c-3624496436ca.png)

This is what you were talking about
![screenshot 39](https://user-images.githubusercontent.com/24369463/34440892-2f48c9d6-ecda-11e7-9690-d06cef223338.png)
